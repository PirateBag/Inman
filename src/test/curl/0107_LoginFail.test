
===  JSON  ===
{ "userName" : "fred-PuPuHead",
  "password" : "dilban"
}
