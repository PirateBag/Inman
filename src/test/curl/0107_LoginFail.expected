{
  "token" : "No Token",
  "status" : "No Token"
}